### DISCUSSION TITLE
Copilot completely broken in VS2022

### DISCUSSION THREAD

**Discussion by Rawalanche** [community/community#79835] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

After one of the latest updated to either VS2022 or Copilot, the extension has became **completely unusable**, se the video:
https://www.youtube.com/watch?v=JoBb5WG1HB8

Starting typing will make copilot suggestion appear only for fraction of second, and the suggestion disappears before I can even reach TAB key to confirm it. It becomes replaced with intellisense autocomplete list, which REMOVES the suggestion.

If I manage to somehow accept the suggestion in time, it autofills completely different suggestion than the one that was displayed. 

Literally only way to "accept" the suggestion is to type the ENTIRE suggestion out by myself.

This is not customized VS2022. This is VS2022 with factory settings, only difference from factory state is the presence of Copilot plugin.

How two Microsoft products can interoperate in such a poor manner is beyond me.

**Comment by Rawalanche** [comment, marked as answer]
It seems to be caused by "Disable Aggressive Member List" option being set to False by default, causing the member list to overwrite Copilot suggestions at all time. So essentially Copilot doesn't work with VS2022 factory defaults.

**Reply by ayim** [reply]
@Rawalanche glad you got this working - I've filed a bug here to track on the Visual Studio side.

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
aayim@microsoft.com
